WireDomain/Sources/WireDomain/Repositories/ModelMappings/FeatureConfigModelMappings.swift

```diff
@@ -0,0 +1,99 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireAPI
+import WireDataModel
+
+extension WireAPI.MLSFeatureConfig {
+
+    func toDomainModel() -> WireDataModel.Feature.MLS.Config {
+        .init(
+            protocolToggleUsers: Array(protocolToggleUsers),
+            defaultProtocol: defaultProtocol == .mls ? .mls : .proteus,
+            allowedCipherSuites: allowedCipherSuites.map {
+                .init(rawValue: $0.rawValue)!
+            },
+            defaultCipherSuite: .init(rawValue: defaultCipherSuite.rawValue)!,
+            supportedProtocols: Set(supportedProtocols.map {
+                .init(rawValue: $0.rawValue)!
+            })
```

Comment:
just to understand, we transform supportedProtocols to supportedProtocols, it's too different types any risk to get nil here?

Reply:
I will provide an exhaustive switch to avoid this, so if we add a new case they will be a compilation error. thanks

Reply:
fixed thx